code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -170,15 +175,17 @@
 	if(!summoner) return
 	if(loc == summoner)
 		forceMove(get_turf(summoner))
-		new /obj/effect/temp_visual/guardian/phase(loc)
+		if(!stealthy_deploying)
+			new /obj/effect/temp_visual/guardian/phase(get_turf(src))
```

Comment:
just save the turf you are teleporting to and use that to initialise the effect. no need for 2 get turfs that point to the same location

Reply:
Done